tests/test_deprecations.py

```diff
@@ -8,13 +8,9 @@
 
 # New modules
 import rasterio
-from rasterio import windows
```

Comment:
Yeah I see that now. I was worried about isort doing something wrong like that. Need to revert this file it looks like.